### PR TITLE
fix: properly truncate long versions in install button

### LINKED ISSUE #6022: package manager: the install buttons wrap with long version numbers
## Describe the bug

with our long 1.0.0-preview.GUID versions, we get a ugly text wrapping in the button.

Screen Shot 2021-02-26 at 4 36 13 PM

### PR DESCRIPTION
**Pull request by benbrown**
## Description

Truncates the long version number in the package manager UI.

Also adds the untruncated version in the package description.

Full untruncated version list is available in the dropdown of the button.

Screen Shot 2021-03-01 at 5 15 38 PM

## Task Item

fixes #6022

### CODE REVIEW

**Review by a-b-r-o-w-n — DISMISSED** [review]

File: extensions/packageManager/src/pages/Library.tsx [hunk 1]
```diff
@@ -724,22 +724,68 @@ const Library: React.FC = () => {
...
+                            maxWidth: 80,
+                            textOverflow: 'ellipsis',
+                            overflow: 'hidden',
+                            whiteSpace: 'nowrap',
+                            display: 'inline-block',
+                          }}
+                          title={selectedVersion}
+                        >
+                          {selectedVersion}
+                        </span>{' '}
```
Comment (srinaath) [inline extensions/packageManager/src/pages/Library.tsx:738]: Why this blank space here? Can we use padding or margin to get this extra space

Comment (hatpick) [reply extensions/packageManager/src/pages/Library.tsx:738]: ^ you can also use &nbsp;

Comment (benbrown) [reply extensions/packageManager/src/pages/Library.tsx:738]: you know, the prettier plugin inserted those! Yes, I will use an nbsp.

Comment (benbrown) [reply extensions/packageManager/src/pages/Library.tsx]: done

**Review by a-b-r-o-w-n — APPROVED** [review]

**Review by srinaath — APPROVED** [review]

**Pull request merged**